addons/project_todo/static/src/scss/todo.scss

```diff
@@ -58,6 +55,12 @@
     .oe_description {
         overflow-x: hidden;
     }
+
+    .o_task_state_widget .o_status{
+        width: $font-size-base * 1.36;
+        height: $font-size-base * 1.36;
+        margin-top: -0.5px;
```

Comment:
code duplication is occurring. [link](https://github.com/odoo/odoo/blob/master/addons/project/static/src/scss/project_widgets.scss#L67)